pkg/abaputils/addonDescriptor.go

```diff
@@ -29,6 +29,7 @@ type Repository struct {
 	Name                string `json:"name"`
 	Tag                 string `json:"tag"`
 	Branch              string `json:"branch"`
+	CommitID            string `json:"commitID"`
```

Comment:
Just wondering: Is there any special check for `CommitID` that would be re-usable on pkg level?

Reply:
What checks do you have in mind?

Reply:
Thinking of other steps that are making use of the addonDescriptor: Checking the repository struct, e.g. if all elements contain a Name/Branch/CommitID.. There will be a check needed anyway to determine if all repository objects contain a name/branch. But I see the use case differs here